src/NetCore/Westwind.Globalization.AspnetCore/Localizers/DbResViewLocalizer.cs

```diff
@@ -78,7 +78,7 @@ public DbResViewLocalizer(IHtmlLocalizerFactory localizerFactory, IWebHostEnviro
         /// <inheritdoc />
         public LocalizedString GetString(string name, params object[] values) => _localizer.GetString(name, values);
 
-#if NET5_0
+#if NET5_0 || NET6_0
```

Comment:
Since we're removing support for net5.0, you can just drop support here as well.